package.json

```diff
@@ -1,78 +1,79 @@
-  "main": "index.js",
-  "scripts": {
-    "start": "npm run server & npm run client",
-    "server": "node --harmony_destructuring --harmony_rest_parameters server/index.js",
-    "client": "webpack-dev-server --inline --watch",
-    "test": "karma start",
-    "codecov": "npm codecov < coverage/lcov/lcov.info",
-    "validate": "eslint --ext .js,.ts --ignore-path .gitignore ./",
-    "build": "webpack --progress --profile"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+ssh://git@github.com/garage-it/SmartHouse-frontend.git"
-  },
-  "keywords": [
-    "smart",
-    "house",
-    "iot"
-  ],
-  "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/garage-it/SmartHouse-frontend/issues"
-  },
-  "homepage": "https://github.com/garage-it/SmartHouse-frontend#readme",
-  "engines": {
-    "node": ">=5"
-  },
-  "devDependencies": {
-    "babel-core": "^6.7.4",
-    "babel-eslint": "^6.0.2",
-    "babel-loader": "^6.2.4",
-    "babel-plugin-angular2-annotations": "^5.0.0",
-    "babel-plugin-transform-class-properties": "^6.6.0",
-    "babel-plugin-transform-decorators-legacy": "^1.3.4",
-    "babel-plugin-transform-flow-strip-types": "^6.7.0",
-    "babel-preset-es2015": "^6.6.0",
-    "babel-register": "^6.7.2",
-    "body-parser": "^1.15.0",
-    "clean-webpack-plugin": "^0.1.8",
-    "core-js": "^2.2.1",
-    "eslint": "^2.6.0",
-    "eslint-config-airbnb": "^6.2.0",
-    "eslint-loader": "^1.3.0",
-    "eslint-plugin-babel": "^3.1.0",
-    "express": "^4.13.4",
-    "html-webpack-plugin": "^2.14.0",
-    "isparta": "^4.0.0",
-    "isparta-loader": "^2.0.0",
-    "jasmine": "^2.4.1",
-    "karma": "^0.13.22",
-    "karma-chrome-launcher": "^0.2.3",
-    "karma-coverage": "^0.5.5",
-    "karma-jasmine": "^0.3.8",
-    "karma-mocha-reporter": "^2.0.0",
-    "karma-phantomjs-launcher": "^1.0.0",
-    "karma-sourcemap-loader": "^0.3.7",
-    "karma-webpack": "^1.7.0",
-    "phantomjs-prebuilt": "^2.1.6",
-    "raw-loader": "^0.5.1",
-    "source-map-loader": "^0.1.5",
-    "webpack": "^1.12.14",
-    "webpack-dev-server": "^1.14.1",
-    "webpack-merge": "^0.9.0"
-  },
-  "dependencies": {
-    "angular2": "^2.0.0-beta.14",
-    "codecov": "^1.0.1",
-    "core-js": "^2.2.2",
-    "reflect-metadata": "^0.1.3",
-    "rxjs": "^5.0.0-beta.5",
-    "socket.io": "^1.4.5",
-    "zone.js": "^0.6.10"
-  }
-}
+{
+  "name": "smart-house-frontend",
+  "version": "0.1.0",
+  "description": "Smart House",
+  "main": "index.js",
+  "scripts": {
+    "start": "npm run server & npm run client",
+    "server": "node --harmony_destructuring --harmony_rest_parameters server/index.js",
+    "client": "webpack-dev-server --inline --watch",
+    "test": "karma start",
+    "codecov": "npm codecov < coverage/lcov/lcov.info",
+    "validate": "eslint --ext .js,.ts --ignore-path .gitignore ./",
+    "build": "webpack --progress --profile"
+	"protractor e2e testing": "webdriver-manager update & webdriver-manager start & protractor conf.js"

```

Comment:
!fix rename task to something simpler (without whitespaces), e.g. "test:e2e", add missing colon